src/chain/payout.rs

```diff
@@ -43,15 +43,18 @@ pub async fn payout(
         let block_number = current_block_number(&client, &chain.metadata, &block).await?;
         let balance = order.balance(&client, &chain, &block).await?; // TODO same
         let loss_tolerance = 10000; // TODO: replace with multiple of existential
-        let manual_intervention_amount = 1000000000000;
+                                    // let manual_intervention_amount = 1000000000000;
```

Comment:
```suggestion
        // let manual_intervention_amount = 1000000000000;
```

Reply:
```suggestion
                                    // TODO: add upper limit for transactions that would require manual intervention
                                    // just because it was found to be needed with non-crypto trade, who knows why?
```